src/app/components/cds/media-cards/Card.js

```diff
@@ -37,7 +38,7 @@ const Card = ({
       onMouseLeave={handleMouseLeave}
     >
       <MaybeLink to={cardUrl}>
-        <div className={styles.cardContent}>
+        <div className={`${styles.cardContent} ${className}`}>
```

Comment:
FYI @caiosba , super nit here but, you can add a conditional classname this way https://github.com/meedan/check-web/blob/develop/src/app/components/cds/buttons-checkboxes-chips/ButtonMain.js#L27

which will avoid the space if the classname is not provided

Reply:
Will do!

Reply:
Done, thanks for the tip!